domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/mapAssertions.kt

```diff
@@ -13,6 +13,11 @@ import ch.tutteli.atrium.translations.DescriptionBasic
 import ch.tutteli.atrium.translations.DescriptionCollectionAssertion
 import ch.tutteli.atrium.translations.DescriptionMapAssertion
 
+fun <K, V> _containsKey(plant: AssertionPlant<Map<K, V>>, key: K): Assertion = AssertImpl.builder.descriptive
+    .withTest { plant.subject.containsKey(key) }
+    .withDescriptionAndRepresentation(DescriptionMapAssertion.MAP_CONTAINS, key.toString())
```

Comment:
please don't use `toString()` otherwise error reporting is misleading. For instance: 
``` 
assert(mapOf(1 to "hello", 3 to "world")).containsKey(2)
```
would report:
```
assert: {1="hello", 3="world"}
- contains key: "2" 
```
Which makes it look like one inadvertently passed `"2"` instead of `2`